### DISCUSSION TITLE
Text Highlight Colors

### DISCUSSION THREAD

**Discussion by isperval** [Adventech/sabbath-school-android#539] [Ideas]
It will be possible to use more subdued colors in the readers' text highlights?
Yellow is perfect, but orange, green and blue I think could be more subdued. There are screens that tend to darken the colors and that makes it difficult to read.
![Screenshot_20230202-083634](https://user-images.githubusercontent.com/124278398/216369689-a50c7929-44e9-4047-9397-d839428bc703.png)

**Comment by TinasheMzondiwa** [comment]
We could try adding functionality to choose a custom color
